app/views/admin/tags/show.html.erb

```diff
@@ -5,7 +5,7 @@
   </div>
 
   <div class="ml-auto">
-    <a href="/t/<%= @tag.name %>">View</a>
+    <a href="/t/<%= @tag.name %>" class="crayons-btn crayons-btn--outlined">View</a>
```

Comment:
`<%= tag_path(@tag.name) %>`

would also be nice if this opened in a separate tab, not to disrupt the editing flow, now that this is essentially an edit page. Something like `target="_blank" rel="noopener noreferer"`